vendor/engines/bulk_email/app/views/bulk_email/bulk_email/_results.html.haml

```diff
@@ -4,6 +4,12 @@
   - if params[:return_path].present?
     = hidden_field_tag :return_path, params[:return_path]
 
+  - if params[:default_text].present?
```

Comment:
To clarify, I am making changes to display the canned text and making that canned text dynamic though not passed in through hidden params, verbatim.